packages/cli/src/lib/bundler/bundle.ts

```diff
@@ -42,7 +42,6 @@ export async function buildBundle(options: BuildOptions) {
     isDev: false,
     baseUrl: resolveBaseUrl(options.frontendConfig),
   });
-  const compiler = webpack(config);
```

Comment:
Is this needed or just a refactor? 😁

Reply:
It was removed in the refactor yes, the compiler is now executed in the callback function so we don't need to store it in a variable anymore.